apps/mobile/src/app/wallet/secure-your-wallet.tsx

```diff
@@ -0,0 +1,91 @@
+import { useRef } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import CircleQuestionMark from '@/assets/circle-questionmark.svg';
+import Lock from '@/assets/lock.svg';
+import { Button } from '@/components/button';
+import { SkipSecureWalletModal } from '@/components/secure-your-wallet/skip-secure-wallet-modal';
+import { useToastContext } from '@/components/toast/toast-context';
+import { ToastType } from '@/components/toast/types';
+import { TransText } from '@/components/trans-text';
+import { APP_ROUTES } from '@/constants';
+import { BottomSheetModal } from '@gorhom/bottom-sheet';
+import { t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+import { useRouter } from 'expo-router';
+
+import { Box, Theme, TouchableOpacity } from '@leather.io/ui/native';
+
+export default function () {
+  const { bottom } = useSafeAreaInsets();
+  const theme = useTheme<Theme>();
+  const skipSecureWalletModalRef = useRef<BottomSheetModal>(null);
+  const router = useRouter();
+  const toastContext = useToastContext();
+
+  function createWallet() {
+    toastContext.displayToast({ type: ToastType.success, title: 'Wallet added successfully' });
```

Comment:
```suggestion
    toastContext.displayToast({ type: 'success', title: 'Wallet added successfully' });
```

Using a string literal > enum means better ts autocompletion & no import statement needed. Overall better DX imo